docs/paper/dev/misc/internal-code.mdx

```diff
@@ -93,7 +107,6 @@ int dataVersion = Bukkit.getUnsafe().getDataVersion();
 
 :::danger[Parsing the version]
 
-You should not parse the version from the package name of classes. This is because CraftBukkit may not be relocated
-in the future.
+Parsing the version from the package name of classes is no longer possible as of 1.20.5 as Paper stopped relocating the CraftBukkit package. See the [reflection](#reflection) section for more information.
```

Comment:
```suggestion
Parsing the version from the package name of classes is no longer possible as of 1.20.5 as Paper stopped relocating the CraftBukkit package. See the [Reflection](#reflection) section for more information.
```

Reply:
why exactly just here.

Reply:
it's the name of the section

Reply:
eh, its not really done at other places either.